DataFormats/FTLRecHit/interface/FTLUncalibratedRecHit.h

```diff
@@ -11,25 +11,33 @@ class FTLUncalibratedRecHit {
   typedef DetId key_type;
 
   enum Flags {
-          kGood=-1,                 // channel is good (mutually exclusive with other states)  setFlagBit(kGood) reset flags_ to zero 
-          kPoorReco,                // channel has been badly reconstructed (e.g. bad shape, bad chi2 etc.)
-          kSaturated,               // saturated channel
-          kOutOfTime               // channel out of time
+          kGood=-1,   // channel is good (mutually exclusive with other states)  setFlagBit(kGood) reset flags_ to zero
+          kPoorReco,  // channel has been badly reconstructed (e.g. bad shape, bad chi2 etc.)
+          kSaturated, // saturated channel
+          kOutOfTime  // channel out of time
   };
 
   FTLUncalibratedRecHit();
-  FTLUncalibratedRecHit(const DetId& detId, float ampl, float time, float timeError, unsigned char flags = 0);
+  FTLUncalibratedRecHit(const DetId& detId, std::pair <float,float> ampl,
+			std::pair <float,float> time, float timeError, unsigned char flags = 0);
+  FTLUncalibratedRecHit(const DetId& detId, uint8_t row, uint8_t column, 
+			std::pair <float,float> ampl, std::pair <float,float> time,
+			float timeError, unsigned char flags = 0);
```

Comment:
is timeError expected to be fixed/common for the dual-readout case?

Reply:
timeError is passed as a parameter and never used. I'm trying to figure out if it's really needed and eventually remove it in a future PR.